Bitfield.pm

```diff
@@ -94,6 +94,8 @@ sub render_bitfield_core {
             emit "static bitfield_identity identity;";
             emit "static bitfield_identity *get() { return &identity; }";
         } "template<> struct ${export_prefix}identity_$traits_name ", ";";
+        header_ref("Export.h");
```

Comment:
I'm not sure if I like DF structures depending directly on files in DFHack - can that be avoided? Maybe we can just include these headers in DataStatic*.cpp?